bin/run.js

```diff
@@ -391,7 +391,7 @@ function pipeToCoverageService (service, options, child) {
     if (signal) {
       process.kill(process.pid, signal)
     } else if (code) {
-      process.exit(code)
+      console.log('Error piping coverage to ' + service.name)
```

Comment:
I'm wondering if this (and the success message as well) should be console.error... probably no one cares tho.

Will be landed in 5.8, not 6.0.